profile/README.md

```diff
@@ -16,13 +16,14 @@
 
 | Name | Description | 
 |---|---| 
-| [imap_processing](https://github.com/IMAP-Science-Operations-Center/imap_processing) | Software for performing L0->L3 data processing | 
+| [imap_processing](https://github.com/IMAP-Science-Operations-Center/imap_processing) | Software for performing L0->L2 data processing | 
 | [sds-data-manager](https://github.com/IMAP-Science-Operations-Center/sds-data-manager) | Software for building and configuring AWS architecture to support data processing | 
 | [imap-data-access](https://github.com/IMAP-Science-Operations-Center/imap-data-access) | Package and command line utility for users to download, query, and upload data from the IMAP Science Data Center | 
+| [imap_L3_processing](https://github.com/IMAP-Science-Operations-Center/imap_L3_processing) | Software for performing L3 data processing |
```

Comment:
I'd probably move this right next to the `imap_processing` so they are one after the other and grouped that way. But that is just a minor preference and not blocking at all.

Reply:
Good call! I agree